model_bakery/faker_gen.py

```diff
@@ -0,0 +1,18 @@
+from typing import Callable, Dict
+
+from faker import Faker
```

Comment:
This import (and its subsequent behavior) should be resilient to `ImportError`, since we don't want to require users to install `faker` if they are not going to use it.